.github/workflows/build.yml

```diff
@@ -63,6 +63,7 @@ jobs:
           extensions: sockets, curl, zip, ffi
           php-version: ${{ matrix.php }}
           coverage: none
+          ini-values: ${{ matrix.operating-system == 'windows-latest' && 'opcache.enable=0 opcache.enable_cli=0' || '' }}
```

Comment:
So this is the only change needed here. Can you separate this from the rest of the refactor and make that it's own PR?

Reply:
Yes, this is one line PR. The rest code is already on the [other PR](https://github.com/pact-foundation/pact-php/pull/339), which I mentioned in the PR's description.

Because of lacks of reviewers + too many PRs to manage, I usually rebase the later PR on top of the previous PR if they are somehow related, so the tests will pass + I can test them together.

I always mentioned which PRs this PR depend on, and they need to be reviewed and merged first in the description.

Reply:
If this is a one-liner, I don't see how it's helping to

a) couple the refactor with it
b) not just have the one liner in the other PR

Reply:
Removed